website/docs/reference/deploy/environment-import-export.md

```diff
@@ -0,0 +1,82 @@
+
+## Import & Export items
+
+The following items will be exported and imported:
+* feature toggles
+* activation strategies
+* variants
+* enabled/disabled statuses
+* context fields
+* feature tags
+* feature tag types
+* segments
+
+## Export
+
+Export is search-driven based on the results you see in the project toggle search or a global toggle search.
+
+![Search-driven export screen with WYSIWYG mode](/img/export.png)
+
+## Import
+
+Import is a 3 stage process designed to be efficient and error-resistant.
+
+### Import stages
+
+* **upload** - you can upload exported file or copy-paste export data into the code editor
+* **validation** - you will get feedback on any errors or warnings before you do the actual import. This makes sure your feature flags configurations
+are compatible with your target environment.
+* **import** - the actual import that creates a new configuration in the target environment or creates a [change request](../change-requests.md) when the environment has change requests enabled
+
+![Three-staged import process](/img/import.png)
+
+### Import rejection
+
+Import can be rejected when:
+* the exported and imported context fields have conflicting context fields values. If the context fields don't exist in the target environment they will be created.
+* the target Unleash instance doesn't have a matching segment. The actual constraints in the segment can be different. The matching is done by the segment name.
+* the exported strategies include custom strategies that don't exist in the target Unleash instance. The matching is done by the strategy name.
+* the exported feature already exists in another project in the target environment that is not a project that you import the toggle to.
+* the target project and environment have a pending change request (applies to enterprise customers with change requests enabled for a given project and environment).
+
+### Import warnings
+
+Import validation will warn you when:
+* the features you are trying to import are already archived in the target instance. Those features won't be imported again.
+* the strategies you are trying to import are custom and may have different configuration parameters. Those strategies will be imported anyway.
+
+### Required permissions
+
+Import requires those permissions:
+* Update feature toggles (always)
+* Create feature toggles when import will be creating new features
+* Update tag types when import will be introducing new tag types
+* Create context fields when import will be creating new context fields
+* Create activation strategies when import will be creating new activation strategies and change requests are disabled
+* Delete activation strategies when import will be deleting existing activation strategies and change requests are disabled
+* Update variants when import will be updating variants and change requests are disabled
```

Comment:
What happens to the last three when change requests are enabled? Does enabling change requests change any of the other ones?

Also is create/delete activation strategies related to creating/deleting custom strategies or to adding/removing strategies from a feature?

```suggestion
To import features, you will **always** require the **update feature toggles** permission.

Additionally, depending on how the actions your import job would trigger, you may also require any of the following permissions:

* **Create feature toggles**: When the import would create new features
* **Update tag types**: when the import would create new tag types
* **Create context fields**: when the import would create new context fields
* **Create activation strategies**: when the import would add (or is this "create custom activation strategies"? custom or standard?) activation strategies to a feature and change requests are disabled
* **Delete activation strategies**: when import would remove existing activation strategies from a feature and change requests are disabled
* **Update variants**: when the import would update variants and change requests are disabled

When change requests are enabled ....
```

Reply:
I like the suggestion